components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/rabbitmq/RabbitMQInjectHandler.java

```diff
@@ -127,7 +129,7 @@ public boolean invoke(RabbitMQMessage message, String inboundName) {
 
         if (injectingSeq == null || injectingSeq.equals("")) {
```

Comment:
This check shouldn't be there inside the invoke method. The initialization itself should fail in this case. WDYT?

Reply:
Let's move the validation of seq being null at line 136 to the initialization as well. This change will be backward incompatible for the users who have deployed injecting sequence after the inbound ep. So, let's clearly mention this in the description.

Reply:
Done in 4c1a1ffdbc47a41a3590bc017bb57689210f029d